java-openjdk/java18-riscv.patch

```diff
@@ -1,8 +1,28 @@
-Index: jdk18u-jdk-18.0.1-10/make/autoconf/jvm-features.m4
-===================================================================
---- jdk18u-jdk-18.0.1-10.orig/make/autoconf/jvm-features.m4
-+++ jdk18u-jdk-18.0.1-10/make/autoconf/jvm-features.m4
-@@ -308,7 +308,8 @@ AC_DEFUN_ONCE([JVM_FEATURES_CHECK_SHENAN
+diff --git a/make/autoconf/build-aux/config.guess b/make/autoconf/build-aux/config.guess
+index d589529f3..e213e020b 100644
+--- a/make/autoconf/build-aux/config.guess
++++ b/make/autoconf/build-aux/config.guess
```

Comment:
If this is a standard `config.guess`, I would actually prefer to update it to a new version instead of adding & maintaining a custom patch.

Reply:
I'll check later if upstream have added riscv there.